ansible/roles/postgresql/tasks/main.yml

```diff
@@ -1,6 +1,26 @@
 ---
 # roles/postgresql/tasks/main.yml
 
+- name: Add PosgreSQL apt repo
```

Comment:
no, it will only upgrade if the ansible state is `latest` but we always use `present` which won't upgrade once its installed